packages/flutter/test/material/tabs_test.dart

```diff
@@ -3301,8 +3300,9 @@ void main() {
               children: <TestSemantics>[
                 TestSemantics(
                     id: 3,
-                    rect: TestSemantics.fullScreen,
+                    rect: const Rect.fromLTRB(0.0, 0.0, 232.0, 48.0),
                     flags: <SemanticsFlag>[SemanticsFlag.hasImplicitScrolling],
+                    transform: Matrix4.translationValues(0.0, 276.0, 0.0),
```

Comment:
This semantic test is expected to be updated because this PR wraps tabs and tab ndicator in a `Stack` widget (divider), which updates the layout to be within the `Stack`.


### Before
| Indicator is aligned with the tab | Inspector view |
| --------------- | --------------- |
| <img src="https://user-images.githubusercontent.com/48603081/236506506-4a618e7a-287b-4a8f-89b3-57244f181241.png" /> | <img src="https://user-images.githubusercontent.com/48603081/236506563-10706a43-acb7-4fdb-b0c0-44fcedf41d31.png" /> |



| Indicator is aligned with the tab after introducing the `Stack` Widget |  Inspector view |
| --------------- | --------------- |
| <img src="https://user-images.githubusercontent.com/48603081/236506647-3bd4b6e9-c388-4a48-98f9-c3715093624e.png" /> | <img src="https://user-images.githubusercontent.com/48603081/236506657-0cb99a83-0709-4382-b1d1-2689c9a58e8f.png" /> |